critiquebrainz/attributes/default.rb

```diff
@@ -0,0 +1,6 @@
+default['critiquebrainz']['user'] = 'critiquebrainz'
+default['critiquebrainz']['group'] = 'critiquebrainz'
+default['critiquebrainz']['home'] = '/home/' + default['critiquebrainz']['user']
+
+default['firewall']['allow_ssh'] = true
+default['critiquebrainz']['open_ports'] = [80, 443]
```

Comment:
Kinda weird that these are configurable but there's no guarantee they'll match what's in https://github.com/metabrainz/chef-cookbooks/pull/36/files#diff-047205e076b909ca49a5edf9d5e04803R2

So I guess it should use these ports in the nginx template too.